client/src/components/QnA/Modals/AddNewAnswer.css

```diff
@@ -0,0 +1,99 @@
+.answer-modal {
```

Comment:
Awesome to see an example of a successful modal. I may  have to do such a thing for the images. Very cool

Reply:
Thanks. Doing the CSS for Modal gave me so much pain.